.gitignore

```diff
@@ -19,9 +19,6 @@ Thumbs.db
 images/*.pdf
 
 /web
-Makefile
-css/cssmenu.css
-*.min.*
```

Comment:
We currently don't generate any `min` files, but if we ever start to do so again, they should be moved into `.generated` anyways.